api/tests/integration/infrastructure/utils/csv/certification-results/get-division-certification-results-csv_test.js

```diff
@@ -52,41 +56,86 @@ describe('Integration | Application | UseCases | certification-results | get-div
       });
     });
     context('when certification has been rejected automatically', function () {
-      it('should return correct csvContent with automatically rejected comment for organization', async function () {
-        // given
+      context('when the reason is insufficient correct answers', function () {
+        it('should return correct csvContent with automatically rejected comment for organization', async function () {
```

Comment:
it('should return correct comment for organization in csvContent ', async function () {